<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="girish-u" failures="1" tests="7" name="Test" time="36.918" errors="0" timestamp="2021-01-21T17:22:23 IST">
  <testcase classname="docker.parallelexecution2" name="test3" time="0.01"/>
  <testcase classname="docker.parallelexecution2" name="test4" time="0.001"/>
  <testcase classname="docker.parallelexecution" name="test1" time="8.792">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="docker.parallelexecution" name="test1" time="11.268">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="docker.parallelexecution" name="test1" time="7.744">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="docker.parallelexecution" name="test1" time="8.461">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at docker.parallelexecution.test1(parallelexecution.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="docker.parallelexecution" name="test2" time="0.001"/>
</testsuite> <!-- Test -->
